code/modules/ninja/suit/mask.dm

```diff
@@ -10,130 +10,9 @@ Contents:
 
 
 
-/obj/item/clothing/mask/gas/voice/space_ninja
+/obj/item/clothing/mask/gas/space_ninja
 	name = "ninja mask"
 	desc = "A close-fitting mask that acts both as an air filter and a post-modern fashion statement."
 	icon_state = "s-ninja"
 	item_state = "s-ninja_mask"
-	vchange = 1
 	strip_delay = 120
-
-/obj/item/clothing/mask/gas/voice/space_ninja/speechModification(message)
```

Comment:
I use broken as a generalized term for the issues they have. Mainly broken as in balance wise in terms of invisibility.
The mask itself works fine. Leave it be.